File: FizzBuzz.js
/**
 * FizzBuzz program written with javascript.
 * 
 * Author - Riley Volk
 * Date - 02.11.2020
 * In collaboration with no one!
 */

let FizzBuzz = function() {
    let array = []

    for(i = 1; array.length <= 100; i++) {
        if (i % 15 == 0) {
            array.push("FizzBuzz")
        } else if (i % 5 == 0) {
            array.push("Buzz")
        } else if (i % 3 == 0) {
            array.push("Fizz")
        } else {
            array.push(i)
        }
    }

    return array
}

console.log(FizzBuzz())

File: FizzBuzz.py
'''
Writing FizzBuzz with Python.

author - Riley Volk
date - 02.02.2020
'''
# FizzBuzz Class
class FizzBuzzClass:
    def FizzBuzz(x = 1):
        while x <= 100:
            if x % 15 == 0:
                print("FizzBuzz")
            elif x % 5 == 0:
                print("Buzz")
            elif x % 3 == 0:
                print("Fizz")
            else:
                print(x)
            x += 1

FizzBuzzClass.FizzBuzz()
